app/src/main/res/values/strings.xml

```diff
@@ -55,8 +56,8 @@
     </string-array>
     
     <plurals name="reward">
-        <item quantity="one">You\'ve been a great guide &#8212; please accept this gold coin.</item>
-        <item quantity="other">You\'ve been a great guide &#8212; please accept these %d gold coins.</item>
+        <item quantity="one">You\'ve been a great guide &#8212; where will you take me next?</item>
+        <item quantity="other">You\'ve been a great guide, thank you!</item>
```

Comment:
Could put %d back in here, something like "we visited %d countries on this trip, that was fun!"